sources/Renderer/Direct3D12/D3D12RenderContext.cpp

```diff
@@ -267,7 +270,7 @@ void D3D12RenderContext::CreateWindowSizeDependentResources(const VideoModeDescr
             swapChainDesc.SampleDesc.Quality    = 0;
             swapChainDesc.BufferUsage           = DXGI_USAGE_RENDER_TARGET_OUTPUT;
             swapChainDesc.BufferCount           = numFrames_;
-            swapChainDesc.Scaling               = DXGI_SCALING_NONE;
+            swapChainDesc.Scaling               = DXGI_SCALING_STRETCH;
```

Comment:
What's the benefit of this scaling option?

Reply:
See my extended comments below. This is only necessary if we want to allow D3D clients to resize the back buffers to be smaller than the size of the HWND their attached to.

Reply:
I'm thinking about unnecessary back buffer filtering. No scaling could potentially allow the simplest form of blitting the back buffer to the screen. With stretching, some form of sampling must be introduced which seems unnecessary for a back buffer that's supposed to have the exact same size as the client area.

Reply:
Again, this allows a client to use a back buffer that's smaller than the HWND but still fills the HWND. The Microsoft documentation doesn't make clear whether turning this on imposes a performance penalty when the HWND and back buffer are the same size and there's no need to stretch. Stretch seems to be the default on D3D11 so I'm guessing it does not.